test/SlowTests/Corax/RavenDB-21632.cs

```diff
@@ -0,0 +1,69 @@
+using System.Linq;
+using FastTests;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Session;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Corax;
+
+public class RavenDB_21632 : RavenTestBase
+{
+    public RavenDB_21632(ITestOutputHelper output) : base(output)
+    {
+    }
+    
+    [Fact]
+    public void CoraxFacetWillNotCountNullValuesInRangeFacets()
+    {
```

Comment:
This should be run in both Lucene and Corax to ensure consistency.

Reply:
And please use `RavenFact`